README.md

```diff
@@ -24,8 +24,10 @@ To get details about the system, read our paper in [SOSP '21](https://doi.org/10
 - `catloop` - TCP Socket Loopback LibOS
 - `catmem` - Shared Memory LibOS
 - `catnap` - Linux Sockets LibOS
+- `catnapw` - Windows Winsock LibOS
```

Comment:
There is no `catnapw` artifact in the codebase or environment, except for the GitHub workflow name to differentiate between Linux and Windows.

How about doing this instead?

- `catnap` - Linux Sockets LibOS / Windows Winsock LibOS